src/views/GroupSettings.vue

```diff
@@ -75,8 +75,10 @@
 
 <script>
 import axios from 'nextcloud-axios'
+import { loadState } from 'nextcloud-initial-state'
+import linkToOCS from 'nextcloud-router'
```

Comment:
```suggestion
import {linkToOCS} from 'nextcloud-router'
```

have you actually tested this? :D

Reply:
Nope :D
Thanks for being here ;)